mesonpy/__init__.py

```diff
@@ -919,6 +926,34 @@ def _parse_version_string(string: str) -> Tuple[int, ...]:
         return (0, )
 
 
+def _get_meson_command(
+        meson_tool: str | None = None, *, version: str = _MESON_REQUIRED_VERSION
+    ) -> List[str]:
+    """Return the command to invoke meson.
```

Comment:
I usually leave an empty line between the function description and the extended docstring.